sdk/src/mempool.ts

```diff
@@ -7,7 +7,7 @@ export const MAINNET_MEMPOOL_BASE_PATH = 'https://mempool.space/api/v1';
  * Base path for the testnet Memopool API.
  * @default "https://btc-testnet.gobob.xyz"
  */
-export const TESTNET_MEMPOOL_BASE_PATH = 'https://mempool.space/testnet4/api/v1';
+export const TESTNET_MEMPOOL_BASE_PATH = 'https://mempool.space/signet/api/v1';
```

Comment:
Should have both no?

Reply:
Reason: There is no support for signet for address validation since testnet and signet address format are same. 
hence in places where we are creating esplora and ordinals client using `addressInfo.network` will not work for signet.

Reply:
Need to write some custom code to support both `testnet` and `signnet` in multiple places to support both.